src/lib.rs

```diff
@@ -17,7 +17,7 @@ use pact::interpreter::{interpret, types::PactType};
 
 mod test;
 
-/// A CENNZnet permission domain module method
+/// A `CENNZnet` permission domain module method
```

Comment:
`CENNZnet` is not a rust type in this case. Can this lint be configured to allow this word or perhaps disabled for now. It's not worth a big chunk of effort